src/i18n/strings/en_EN.json

```diff
@@ -1429,7 +1429,8 @@
     "For help with using %(brand)s, click <a>here</a> or start a chat with our bot using the button below.": "For help with using %(brand)s, click <a>here</a> or start a chat with our bot using the button below.",
     "Chat with %(brand)s Bot": "Chat with %(brand)s Bot",
     "Bug reporting": "Bug reporting",
-    "If you've submitted a bug via GitHub, debug logs can help us track down the problem. Debug logs contain application usage data including your username, the IDs or aliases of the rooms or groups you have visited, which UI elements you last interacted with, and the usernames of other users. They do not contain messages.": "If you've submitted a bug via GitHub, debug logs can help us track down the problem. Debug logs contain application usage data including your username, the IDs or aliases of the rooms or groups you have visited, which UI elements you last interacted with, and the usernames of other users. They do not contain messages.",
+    "If you've submitted a bug via GitHub, debug logs can help us track down the problem. ": "If you've submitted a bug via GitHub, debug logs can help us track down the problem. ",
```

Comment:
Mind the whitespace character after the dot. This can be ignored on Weblate by translators.